### ISSUE TITLE
Spurious forces for PdAg system

### ISSUE THREAD

**Issue by WeckmanTimo** [dftd4/dftd4#224]
**Describe the bug**
When using the dftd4 with a GPAW for calculations with a PdAg alloy, the DFTD4 (3.4.0, 3.6.0) returns sometimes (reproducibly, but seemingly at random) very large forces.

**To Reproduce**
Here is an example script to reproduce the issue (should occur at the initial optimization step):

from ase.build import fcc111
from ase.optimize import BFGS
from ase.constraints import FixAtoms
from ase.calculators.mixing import SumCalculator
from gpaw import GPAW, PW
from dftd4.ase import DFTD4

surface = fcc111('Ag', size=(3,3,5), a=4.05)
surface.center(vacuum=10, axis=2)
surface[40].symbol = 'Pd'
surface.set_constraint(FixAtoms([x for x in range(18)]))
calc = GPAW(mode=PW(500), xc='PBE', kpts=(3,3,1), txt='PdAg_output.txt')
calc_sum = SumCalculator([calc, DFTD4(method='PBE')])
surface.calc = calc_sum
BFGS(surface, trajectory='PdAg_opt.traj').run(fmax=0.05)

This calculation results in forces
      Step     Time          Energy         fmax
BFGS:    0 10:34:32     -144.919442      332.8840

while the fmax obtained from the DFT/PBE calculation alone is only 0.35 eV/ang.
These forces veer the optimization off the straigth path to minimum. However, the energy from the DFTD4 calculator appers to be smooth.

**Expected behaviour**
The forces should be relatively smooth and not contain sudden peaks that distort the geometry.

**Comment by marvinfriede** [comment]
Can you provide a more minimal example, i.e., a single structure file that can be run with the Fortran program? If the problem occurs in the first optimization step, it may be an issue with the gradient that could be reproducible with the standalone D4 program. That would greatly help debugging.

**Comment by WeckmanTimo** [comment]
Yes, since the problem arises with the initial step, the code example works if the GPAW-section are just commented out. I have also attached the traj/xyz files for this structure [PdAg.zip](https://github.com/dftd4/dftd4/files/13331103/PdAg.zip).
